doc/user/content/sql/create-cluster.md

```diff
@@ -105,16 +105,18 @@ _replica_name_ | A name for a cluster replica.
 
 {{% replica-options %}}
 
-{{< note >}}
-If you do not specify an availability zone, Materialize will automatically
-assign the availability zone with the least existing replicas for the
-associated cluster to increase the cluster's tolerance to availability zone
-failure.
-
-To check the availability zone associated with each replica in a cluster, use
-the [`mz_cluster_replicas`](/sql/system-catalog/mz_catalog/#mz_cluster_replicas)
-system table.
-{{< /note >}}
+## Availability Zones and Scheduling
```

Comment:
```suggestion
## Availability zone assignment
```

(I think "assignment" is a bit clearer for the end user, since they're not thinking about the Kubernetes scheduler.)